x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action_dry_run.ts

```diff
@@ -296,7 +296,7 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
-    describe('schedule manual rule run action', () => {
+    describe('@skipInServerless schedule manual rule run action', () => {
```

Comment:
This one need to be also skipped in MKI with `@skipInServerlessMKI`. Manual rule run feature is hidden behind feature flag which are not supported in MKI.

Reply:
`@skipInServerless` is enough to skip in MKI as well, I added extra `@skipInServerlessMKI` tags for visibility, gonna add them here too :)

Reply:
Added